normandy/recipes/templates/admin/field_recipe_list.html

```diff
@@ -0,0 +1,9 @@
+{% if recipes %}
+  <ul>
+    {% for recipe in recipes %}
+      <li><a href="{% url 'admin:recipes_recipe_change' recipe.id %}">{{ recipe.name }}</a></li>
+    {% endfor %}
+  </ul>
+{% else %}
+  <p>Not used in any recipes.</p>
```

Comment:
I like the checkmark suggestion, but I think we can add that later. I'll go with the easy fix for now.